coach/src/navigation/index.cjsx

```diff
@@ -40,7 +40,7 @@ Navigation = React.createClass
 
   onCourseEnded: (notice) ->
     NotificationActions.acknowledge(notice)
-    channel.emit('show.registration', view: 'second-semester-registration')
+    channel.emit('show.second-semester-registration', view: 'second-semester-registration')
 
```

Comment:
this is an unrelated bug fix for the clicking on the second semester link.  I had it as a separate pr but it snuck in here